schemas/io-package.json

```diff
@@ -91,15 +115,40 @@
                 "license": {
                     "description": "License of the software",
                     "type": "string",
-                    "pattern": "(?i)afl-3.0|apache-2.0|artistic-2.0|bsl-1.0|bsd-2-clause|bsd-3-clause|bsd-3-clause-clear|cc|cc0-1.0|cc-by-4.0|cc-by-sa-4.0|wtfpl|ecl-2.0|epl-1.0|epl-2.0|eupl-1.1|agpl-3.0|gpl|gpl-2.0|gpl-3.0|lgpl|lgpl-2.1|lgpl-3.0|isc|lppl-1.3c|ms-pl|mit|mpl-2.0|osl-3.0|postgresql|ofl-1.1|ncsa|unlicense|zlib"
+                    "pattern": "afl-3.0|AFL-3.0|apache-2.0|APACHE-2.0|artistic-2.0|ARTISTIC-2.0|bsl-1.0|BSL-1.0|bsd-2-clause|BSD-2-CLAUSE|bsd-3-clause|BSD-3-CLAUSE|bsd-3-clause-clear|BSD-3-CLAUSE-CLEAR|cc|CC|cc0-1.0|CC0-1.0|cc-by-4.0|CC-BY-4.0|cc-by-sa-4.0|CC-BY-SA-4.0|wtfpl|WTFPL|ecl-2.0|ECL-2.0|epl-1.0|EPL-1.0|epl-2.0|EPL-2.0|eupl-1.1|EUPL-1.1|agpl-3.0|AGPL-3.0|gpl|GPL|gpl-2.0|GPL-2.0|gpl-3.0|GPL-3.0|lgpl|LGPL|lgpl-2.1|LGPL-2.1|lgpl-3.0|LGPL-3.0|isc|ISC|lppl-1.3c|LPPL-1.3C|ms-pl|MS-PL|mit|MIT|mpl-2.0|MPL-2.0|osl-3.0|OSL-3.0|postgresql|POSTGRESQL|ofl-1.1|OFL-1.1|ncsa|NCSA|unlicense|UNLICENSE|zlib|ZLIB"
```

Comment:
Wow, there is really no case-insensitive flag 😕

Reply:
Unfortunately yes, it worked at least in my IDE but it's not officially supported and will crash tests from schemastore